app/src/main/res/values/styles_toolbar.xml

```diff
@@ -18,4 +18,7 @@
         <item name="android:windowActionBarOverlay">false</item>
     </style>
 
+    <style name="ToolbarTitle" parent="@style/TextAppearance.AppCompat.Title">
+        <item name="android:textSize">20sp</item>
```

Comment:
why size 20dp. I don't find material guideline.

Reply:
What should be the size then?

Reply:
@therajanmaurya I found out that the default size for the portrait mode text is 20sp while it shrinks down to 14sp for landscape mode. I have tried to make the size standard by fixing it to 20sp i.e similar to that of portrait mode. What changes should I make?

Reply:
@therajanmaurya Here is the guideline link which says App bar: Title style, Medium 20sp

Reply:
ok